apps/calendar/build/build.js

```diff
@@ -29,11 +29,12 @@ exports.execute = function(options) {
   utils.ensureFolderExists(utils.getFile(options.STAGE_APP_DIR));
 
   dump('Will run rjs optimizer...\n');
-  var optimize = new Promise((accept, reject) => {
+  var optimizejs = new Promise((accept, reject) => {
     requirejs.optimize([config.path], accept, reject);
   });
 
-  optimize.then(() => {
+  optimizejs.then((log) => {
```

Comment:
Also AFAICT the changes here are just logging the result of the call to `requirejs.optimize()`